scripts/ci

```diff
@@ -4,18 +4,17 @@
 
 require('./util/work-from-root')();
 
-var cleanDirectory = require('./util/clean-directory');
+var fsExtra = require('fs-extra');
 var buildProject = require('./util/build-project');
 var npmDo = require('./util/npm-do');
-var fs = require('fs');
 
 var testBuildPath = '.dist-test';
 
-cleanDirectory(testBuildPath);
+fsExtra.removeSync(testBuildPath);
 
 buildProject(testBuildPath, function (buildStatus) {
   if (buildStatus === 0) {
-    var xunitWriteStream = fs.createWriteStream('tests.xml');
+    var xunitWriteStream = fsExtra.createWriteStream('tests.xml');
```

Comment:
I feel that that obfuscates where things come from. I'd prefer we use builtins wherever possible, and `fsExtra` for things only supplied by `fsExtra`